src/Libraries/RevitNodes/Elements/CurveElement.cs

```diff
@@ -123,19 +123,18 @@ protected static void setCurveMethod(Autodesk.Revit.DB.CurveElement ce, Curve c)
 
         #endregion
 
+        /// <summary>
+        /// Empty method - we don't want to tessellate curves automatically
+        /// but it seems we need this method to correctly import this library into Dynamo.
```

Comment:
It's already empty in the `abstract class Element`, is it because it's abstract method there so child has to implement?

Reply:
I'm not sure, compilation is fine, but when you attempt to import the child class into the VM, it fails without this method implemented, might be the VM is more strict than c# regarding abstract classes?

Reply:
Could be. Just feeling strange to have a public method not doing anything but also not abstract..